api/envoy/config/core/v3/protocol.proto

```diff
@@ -171,6 +172,9 @@ message AlternateProtocolsCacheOptions {
 
   // Allows pre-populating the cache with entries, as described above.
   repeated AlternateProtocolsCacheEntry prepopulated_entries = 4;
+
+  // Optional list of hostnames suffixes for which Alt-Svc entries can be shared.
```

Comment:
WDYT of adding an example.com based example, along with matches?
also I'm not clear if foo.c.youtube.com would match both .c.youtube.com and .youtube.com or only the former so let's make that clear here.

Reply:
Good idea. PTAL!